fabric-common/test/DiscoveryService.js

```diff
@@ -256,6 +256,31 @@ describe('DiscoveryService', () => {
 			discovery.build(idx, {local: true, interest: interest});
 			should.exist(discovery._action);
 			should.exist(discovery._payload);
+			sinon.assert.calledWith(FakeLogger.debug, '%s - adding chaincodes/collections query');
+		});
+		it('should build with an interest option', () => {
```

Comment:
There are four tests with the same description. They should probably be unique so it is obvious which test produced any failure.